apps/torus-cache/src/data.ts

```diff
@@ -51,49 +52,96 @@ export function getStakeFromDataStringified() {
   return stakeFromDataStringified.data;
 }
 
-const updateStakeFrom = async (api: ApiPromise, lastBlock: LastBlock) => {
-  try {
-    const stakeFrom = await queryStakeIn(api);
-    stakeFromData = {
-      total: stakeFrom.total,
-      perAddr: Object.fromEntries(stakeFrom.perAddr),
-      atBlock: BigInt(lastBlock.blockNumber),
-      atTime: new Date(),
-    };
-    log(`StakeFrom data updated for block ${lastBlock.blockNumber}`);
-  } catch (error) {
+/**
+ * Updates the stake data from the chain
+ * @param api API instance for blockchain queries
+ * @param lastBlock Latest block information
+ */
+export const updateStakeFrom = async (
+  api: ApiPromise,
+  lastBlock: LastBlock,
+) => {
+  const [error, stakeForm] = await tryAsync(queryStakeIn(api));
+  if (error !== undefined) {
     log(
       `Error updating StakeFrom data for block ${lastBlock.blockNumber}:`,
       error,
     );
+    return;
   }
+  stakeFromData = {
+    total: stakeForm.total,
+    perAddr: Object.fromEntries(stakeForm.perAddr),
+    atBlock: BigInt(lastBlock.blockNumber),
+    atTime: new Date(),
+  };
+  log(`StakeFrom data updated for block ${lastBlock.blockNumber}`);
 };
 
-const updateStakeOut = async (api: ApiPromise, lastBlock: LastBlock) => {
-  try {
-    const stakeOut = await queryStakeOut(api);
-    stakeOutData = {
-      total: stakeOut.total,
-      perAddr: Object.fromEntries(stakeOut.perAddr),
-      atBlock: BigInt(lastBlock.blockNumber),
-      atTime: new Date(),
-    };
-    log(`StakeOut data updated for block ${lastBlock.blockNumber}`);
-  } catch (error) {
+/**
+ * Updates staking data for outgoing stake allocations from the blockchain
+ *
+ * Queries the current stake-out information from the blockchain and updates
+ * the local stakeOutData object with the total stake, per-address breakdown,
+ * and current block/time information.
+ *
+ * @param api - Polkadot API instance used to query blockchain data
+ * @param lastBlock - Latest block information including block number
+ * @returns Promise that resolves when the operation completes
```

Comment:
Please remove as many redundant comments and documentation as possible, such as "Promise that resolves when the operation completes." Of course, it returns a promise that resolves when the operation completes; it's an async function, and all of them do this.